web/rest/client/features/provider/faxesInOut/postFaxesInOut.feature

```diff
@@ -0,0 +1,109 @@
+Feature: Create faxes in outs
+  In order to manage faxes in out
+  As a client admin
+  I need to be able to create them through the API.
+
+  @createSchema
+  Scenario: Create a fax out
+    Given I add Company Authorization header
+     When I add "Content-Type" header equal to "multipart/form-data; boundary=----IvozProviderFormBoundaryFUBrG71LG0e8DuZ8"
+      And I add "Accept" header equal to "application/json"
+      And I send a "POST" multipart request to "/faxes_in_outs" with body:
+    """
+----IvozProviderFormBoundaryFUBrG71LG0e8DuZ8
+Content-Disposition: form-data; name="brand"
+
+      {
+          "fax": 1,
+          "src": "34688888888",
```

Comment:
`src` field should also be readonly for Faxes Out, as it's assigned from configured outgoingDDI field